packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/container.tsx

```diff
@@ -243,6 +281,7 @@ const CompleteResetPasswordContainer = ({
 
         // TODO add frontend Glean event for successful reset?
         notifyClientOfSignin(accountResetData);
+        await maybeNotifyOAuthClient(accountResetData);
```

Comment:
Would suggest we remove this line as it would conflict with https://github.com/mozilla/fxa/pull/17989/files and result in the keyFetchToken being used twice

Reply:
Or Dan's patch should be backed out - depends at which point we expect users to be signed in to Sync

Reply:
I don't have much of an opinion. Either could work. The only edge case I would consider is 2FA. If required, I don't think it will have been entered at this point, and therefore the session is technically unverified. 

I'm actually not the most familiar with all the ins and outs of this flow, so feel to correct me if I am off base about this.

Reply:
Leaving your patch in and removing this line makes the most sense to me for 2FA. Thanks!